repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/nilanding.ftl

```diff
@@ -8,21 +8,21 @@
         <h1 data-uipath='ps.document.title' style="margin: 0px 0px 25px">${document.title}</h1>
         <#outputformat "undefined">${document.mainContent.content}</#outputformat>
 
-        <section class="push-double--bottom" style="margin: 40px 0px 40px">
+        <section class="push-double--bottom" style="margin: 40px 0px 40px" title="Advice">
             <h2>${document.adviceTitle}</h2>
             <#outputformat "undefined">${document.adviceContent.content}</#outputformat>
             <a class="niFormBtn" title="${document.adviceForm.text}" href="<@hst.link hippobean=document.adviceForm.resource/>" onClick="logGoogleAnalyticsEvent('Download advice form','Indicator','${document.adviceForm.resource.filename}');"><@fmt.message key="headers.adviceForm"/></a>
             <span class="fileSize">(<@formatFileSize bytesCount=document.adviceForm.resource.length/>)</span>
         </section>
 
-        <section class="push-double--bottom">
+        <section class="push-double--bottom" title="Add">
```

Comment:
Could we please have `title` replaced with `data-uipath` for the purpose of targeting the element from acceptance tests (see comment above for `title="popular-topics"`)?